test/common/testlib.py

```diff
@@ -835,10 +796,7 @@ def setUp(self):
 
         def sitter():
             if opts.sit and not self.checkSuccess():
-                if self._outcome:
-                    [traceback.print_exception(*e[1]) for e in self._outcome.errors if e[1]]
-                else:
-                    self.currentResult.printErrors()
+                [traceback.print_exception(*e[1]) for e in self._outcome.errors if e[1]]
```

Comment:
@martinpitt, on IRC you said that we are missing some backtraces because we only print them with `-s`, and that this PR fixes that. But this change here doesn't change the logic, does it?

Reply:
Upps, the next commit probably does.

Reply:
It does, this line is now run even without `opts.sit()`.